packages/tealdeer/Cargo.toml.patch

```diff
@@ -1,11 +1,11 @@
---- ./Cargo.toml.bak	2022-02-19 02:03:29.085335654 +0530
-+++ ./Cargo.toml	2022-02-19 02:04:45.461335659 +0530
-@@ -20,7 +20,7 @@
- 
+--- ./Cargo.toml.bak    2022-11-10 08:41:55.427076595 +0530
++++ ./Cargo.toml        2022-11-10 08:42:20.667076577 +0530
+@@ -21,7 +21,7 @@
+
  [dependencies]
- ansi_term = "0.12.0"
+ anyhow = "1"
 -app_dirs = { version = "2", package = "app_dirs2" }
 +app_dirs = { version = "1.2.1" }
```

Comment:
Sorry, I misread this patch the first time around. This cargo.toml patch can most likely be dropped also, [this commit](https://github.com/app-dirs-rs/app_dirs2/commit/4d6358485944851afc5972903828744d43d11c36) added support for termux to app_dirs2, so we no longer have to use an older version